BAZProject/MovieDetail/MovieDetailPresenterImp.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  MovieDetailPresenter.swift
+//  BAZProject
+//
+//  Created 1030364 on 17/10/22.
+//  Copyright © 2022 ___ORGANIZATIONNAME___. All rights reserved.
+//
+//  Template generated by Juanpe Catalán @JuanpeCMiOS
+//
+
+import UIKit
+
+final class MovieDetailPresenterImp: MovieDetailPresenter {
+
+    weak private var view: MovieDetailView?
+    var interactor: MovieDetailInteractor?
```

Comment:
Esta propiedad podría ser privada?
```suggestion
    private var interactor: MovieDetailInteractor?
```